invenio_base/__init__.py

```diff
@@ -24,6 +24,129 @@
 
 """Invenio application loader.
 
+Quickstart
+----------
+Invenio-Base is taking advantage of advanced patterns for building Flask
+application. It assumes you already have understanding of
+`patterns for Flask <http://flask.pocoo.org/docs/dev/patterns/>`_.
+
+Dependencies
+~~~~~~~~~~~~
+First we need to install and import dependencies.
+
+.. code-block:: console
+
+   $ mkvirtualenv example
+   (example)$ pip install invenio-base
+
+Now you can create new file ``app.py`` with following imports.
+
+.. include:: ../examples/app.py
+   :start-after: # sphinxdoc-example-import-begin
+   :end-before: # sphinxdoc-example-import-end
+   :literal:
+
+Configuration
+~~~~~~~~~~~~~
+Tell the application factory how to load configuration by creating
+``config_loader`` function that accepts an application instance.
+
+.. include:: ../examples/app.py
+   :start-after: # sphinxdoc-example-config-begin
+   :end-before: # sphinxdoc-example-config-end
+   :literal:
+
+The recommended way is to use Invenio-Config that provides a default
```

Comment:
An explicit link to Invenio-Config documentation home page would be good too.  (In addition to the `create_config_loader` function that you nicely inter-linked below.) 

In general, let's cross-link all the `Invenio-Foo` packages among themselves, to ease the navigation and create a harmonious ecosystem.